mmv1/third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -1826,6 +1825,36 @@ func resourceComputeInstanceUpdate(d *schema.ResourceData, meta interface{}) err
 		}
 	}
 
+	if d.HasChange("can_ip_forward") {
+		err = retry(
+			func() error {
+				instance, err := config.NewComputeClient(userAgent).Instances.Get(project, zone, instance.Name).Do()
+				if err != nil {
+					return fmt.Errorf("Error retrieving instance: %s", err)
+				}
+
+				instance.CanIpForward = d.Get("can_ip_forward").(bool)
+
+				op, err := config.NewComputeClient(userAgent).Instances.Update(project, zone, instance.Name, instance).Do()
```

Comment:
We're starting to get a couple calls to the new Instances.Update in this resource. It will probably be worth finding a pattern to consolidate those soon